### PR TITLE
#48 added cutdown command

### PR DESCRIPTION
**Pull request by DarylDohner**

### CODE REVIEW

File: EosGround/sendCommand.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 from config.config import get_config
+from EosLib.device import Device
 
 import psycopg2
 import datetime
 
-conn_params = get_config('database.ini')  # gets config params
+import os
+
+conn_params = get_config(os.path.join('EosGround', 'config', 'database.ini'))  # gets config params
```
Comment (David-Rey) [inline EosGround/sendCommand.py:12]: Remove EosGround from os.path.join

File: EosGround/sendCommand.py [hunk 2]
```diff
@@ -21,20 +25,49 @@ def send_command():
...
+
+
+def send_cutdown_command():
+    ans = input("Are you sure you want to trigger the cutdown?  (y/n):\t")
+    if ans.lower() == 'y':
+        packet_generate_time = datetime.datetime.now()
+        packet_sender = Device.GROUND_STATION_1
+        packet_type = Type.DATA
+        packet_priority = Priority.DATA
+        packet_destination = Device.MISC_RADIO_1
```
Comment (David-Rey) [inline EosGround/sendCommand.py:43]: Destination should be Device.MISC_ENGINEERING_2

**Review by David-Rey — CHANGES_REQUESTED** [review]

File: EosGround/sendCommand.py (hunk 1 above)
Comment (DarylDohner) [reply EosGround/sendCommand.py:12]: No

File: EosGround/sendCommand.py (hunk 2 above)
Comment (DarylDohner) [reply EosGround/sendCommand.py:43]: good catch

**Review by David-Rey — APPROVED** [review]
looks good

**Pull request merged**